Fix #1790 : Include the file path in get_metadata() UnicodeDecodeErrors (#1791)
pypa/setuptools: changelog.d/1790.change.rst

> Add to the error message the path to the file if a ``UnicodeDecodeError``

Reviewer: ```suggestion
Added the file path to the error message file when a ``UnicodeDecodeError``
```
